easybuild/easyblocks/g/gurobi.py

```diff
@@ -41,15 +42,32 @@
 class EB_Gurobi(Tarball):
     """Support for installing linux64 version of Gurobi."""
 
+    @staticmethod
+    def extra_options(extra_vars=None):
+        """Define extra options for Gurobi"""
+        extra = {
+            'copy_license_file': [True, "Copy license_file to installdir", CUSTOM],
+        }
+        return Tarball.extra_options(extra_vars=extra)
+
+    def __init__(self, *args, **kwargs):
+        """Easyblock constructor, define custom class variables specific to Gurobi."""
+        super(EB_Gurobi, self).__init__(*args, **kwargs)
+
+        self.license_file = 'UNKNOWN'
```

Comment:
why not `None`? You run a check for `is None` below?

Reply:
it doesn't matter what the value is here, because it is always reassigned to `self.cfg['license_file']`.
it's not strictly necessary to initialize it here, but github tests complain if I don't.
I just used 'UNKNOWN' because that's what the other easyblocks do.

Reply:
Actually, it does matter: if `--module-only` is used, we'll now end up with `UNKNOWN` in the module file, because `install_step` is never run.

You can actually use `self.cfg['license_file']` already here, because the easyconfig file is already parsed when the easyblock is initialized (the parsed easyconfig is an input for the easyblock initialization):

```python
self.license_file = self.cfg['license_file']

Reply:
@boegel 
ah good point, I'll fix it.
then the `pgi.py` easyblock has the same problem?